media-plugins/qt-heif-image-plugin/qt-heif-image-plugin-0.3.3.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="qt-heif-image-plugin: Qt plugin for HEIF images"
```

Comment:
delete "qt-heif-image-plugin:" from the description

Reply:
Removed.